src/test/scala/uk/gov/hmrc/test/AkkaMaterializerSpec.scala

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
This `AkkaMaterializerSpec` feels like a testing preference, if thats the case it should be a separate PR and added in for all tests.